### ISSUE TITLE
Enable Fused Winograd by Default

### ISSUE THREAD

**Issue by cancan101** [tensorflow/tensorflow#9339]
Right now [fused Winograd](https://github.com/tensorflow/tensorflow/pull/4901) is [disabled by default](https://github.com/tensorflow/tensorflow/blob/e69f71759adac4a794d5b159358af5253cb243bf/tensorflow/stream_executor/cuda/cuda_dnn.cc#L1970-L1976). This is even though enabling this speeds up models considerable in the 3x3 case (see https://github.com/tensorflow/tensorflow/issues/9322#issuecomment-295779638). What remains as far as issues, etc to get this faster conv enabled by default?

/CC @yangzihao @tfboyd

**Comment by drpngx** [comment]
@tfboyd it might be good to see internally whether we want to accelerate this.

**Comment by tfboyd** [comment]
Working on it.  I found a comment in the code about some failing unit tests and I am following up.  On K80s this makes a big difference.  I am also sending out a note to tensorflow-discuss in the super near future and update the Performance Guide for the near-term.

**Comment by tfboyd** [comment]
This is so close.  Is is submitted internally and I am working to get it synced to github.  Why say anything until it is in master?  Because I am so happy this is finally nearly done.  :-)  I will update and close as soon as I see this in master.

**Comment by tfboyd** [comment]
Done https://github.com/tensorflow/tensorflow/pull/10359  It will most likely NOT be 1.2 but will be in 1.3.

**Comment by tfboyd** [comment]
This is old but I saw it and we had to turn it off again by default in 1.3 but at head and 1.4 going forward everything is all good.  We had issues with a few corner cases used by teams inside Google that would fail.  We worked with NVIDIA to identify all of the possible corner cases and the issue is finally closed.  An FYI if anyone sees this.  We had the same problem with fused batch norm, which will also be the default in a couple of weeks if current progress continues.  With the large amount of production level products using TensorFlow what seems like an easy change can have unforeseen impacts.
